src/test/resources/application.yml

```diff
@@ -164,9 +164,11 @@ bitbucket:
 azure:
   webhook-token: cxflow:1234
   token: xxxx
-  url: https://dev.azure.com/XXXXXX
+  url: https://dev.azure.com/cxflowtestuser
```

Comment:
is that intentionally?

Reply:
Yes, this is to ensure that we use the same organization in ADO.